Angelo/add scribe (#972)
elevenlabs/elevenlabs-docs: fern/docs/pages/cookbooks/speech-to-text/streaming.mdx

# Perform the text-to-speech conversion
transcription = client.speech_to_text.convert_as_stream(

>     model_id="scribe_v1", # "scribe_v1_experimental" is also available

Reviewer: Don't need this as streaming isn't enabled yet. By the time it is I think the experimental changes will have been rolled into v1.

const transcription = await client.speechToText.convertAsStream({
  file: audioBlob,

>   model_id: 'scribe_v1', // Model to use, "scribe_v1_experimental" is also available

Reviewer: Can we apply this to the other instances as well?

```suggestion
  model_id: 'scribe_v1', // 'scribe_v1_experimental' is also available for new, experimental features
```
